inference-engine/src/readers/CMakeLists.txt

```diff
@@ -10,10 +10,12 @@ target_include_directories(${TARGET_NAME} INTERFACE
     "${CMAKE_CURRENT_SOURCE_DIR}/reader_api"
     $<TARGET_PROPERTY:inference_engine,INTERFACE_INCLUDE_DIRECTORIES>)
 
-file(GLOB_RECURSE reader_api_src "${CMAKE_CURRENT_SOURCE_DIR}/reader_api/*.hpp"
-                                 "${CMAKE_CURRENT_SOURCE_DIR}/reader_api/*.h")
+file(GLOB_RECURSE reader_api_hpp "${CMAKE_CURRENT_SOURCE_DIR}/reader_api/*.hpp")
+file(GLOB_RECURSE reader_api_src "${CMAKE_CURRENT_SOURCE_DIR}/reader_api/*.cpp")
```

Comment:
If I remember right reader_api doesn't have any cpp files.

Reply:
I've moved blob_stream.cpp as interface source file for reader API. 

It is needed to resolve the issue with multiple definitions issues with std::ios:base which is used as a base class for BLobStream. BlobStream is exported from IE, and std::ios_base dctors become exported as well and it causes multiple symbols definitions when we link with MSVC static runtime (one symbols from IE.dll because of exported BlobStream/std::ios_base and one from static version of runtime itself)

And this is why I have added `inference_engine_reader_api` as link library